src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCves.js

```diff
@@ -34,9 +34,8 @@ class VulnerabilitiesCves extends Component {
             fetchResource({ ...this.state, page_size: Number.MAX_SAFE_INTEGER, data_format: format, page: 1 });
         payload && payload.then(({ data: response }) => {
             const data = format === 'json' ? JSON.stringify(response) : response;
-            return downloadFile(data, undefined, format);
-        }
-        );
+            return downloadFile(data, ('vulnerability_cves-' + `${new Date().toISOString()}`), format);
```

Comment:
instead of concatenating two strings you can construct one string by putting it all inside the backticks for the filename param.

`` `vulnerability_cves-${new Date().toISOString()}` ``

Reply:
Fixed.